CDCgov/prime-reportstream
# 6299 Use FHIR to HL7 base schema in translation process

This PR adds the use of our new schema when doing the FHIR to HL7 translation in the pipeline.

Test Steps:
1. Check out this branch.
2. Build and run RS.
3. Send the sample HL7 file `prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.hl7` to RS using the following headers:
 a. Content-Type: application/hl7-v2
 b. client: ignore.ignore-full-elr
4. Wait until the data is processed.
5. Open your MS Azure Storage Explorer and view the reports blob container.  Navigate to `batch/ignore.FULL_ELR/`.  Verify that there is a new HL7 file there.
6. Open the new HL7 file and verify that is has the expected segments based on the configuration merged into master.  For reference, the old code would just generate a partial MSH segment only and nothing else.

## Changes
- Use FHIR to HL7 base schema in translation process
- Added new full-elr receiver

## Checklist

### Testing

### Process

## Linked Issues
- #6299

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
